ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -24,7 +24,9 @@ Metadata:
 Mappings:
   NotebookTypes:
     Rstudio:
-      AMIID: "ami-051310d409a32a5aa"  # https://github.com/Sage-Bionetworks-IT/packer-rstudio/tree/v2.0.0
+      # TODO : currently using test AMI, build new AMI once new version of R and Python Client are released
```

Comment:
We can't merge until we build a new AMI that has `SynapseR >= 0.9.78`
https://github.com/Sage-Bionetworks/synapser/tags